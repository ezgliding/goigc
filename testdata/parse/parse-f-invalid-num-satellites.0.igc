F1602310a02
