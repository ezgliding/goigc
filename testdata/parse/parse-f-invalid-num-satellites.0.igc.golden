{
  "Manufacturer": "",
  "UniqueID": "",
  "AdditionalData": "",
  "Date": "0001-01-01T00:00:00Z",
  "FixAccuracy": 0,
  "Pilot": "",
  "Crew": "",
  "GliderType": "",
  "GliderID": "",
  "GPSDatum": "",
  "FirmwareVersion": "",
  "HardwareVersion": "",
  "FlightRecorder": "",
  "GPS": "",
  "PressureSensor": "",
  "CompetitionID": "",
  "CompetitionClass": "",
  "Timezone": 0,
  "Points": null,
  "K": null,
  "Events": null,
  "Satellites": [
    {
      "Time": "0000-01-01T16:02:31Z",
      "Ids": [
        "0a",
        "02"
      ]
    }
  ],
  "Logbook": null,
  "Task": {
    "DeclarationDate": "0001-01-01T00:00:00Z",
    "Date": "0001-01-01T00:00:00Z",
    "Number": 0,
    "Takeoff": {
      "Lat": 0,
      "Lng": 0,
      "Time": "0001-01-01T00:00:00Z",
      "FixValidity": 0,
      "PressureAltitude": 0,
      "GNSSAltitude": 0,
      "IData": null,
      "NumSatellites": 0,
      "Description": ""
    },
    "Start": {
      "Lat": 0,
      "Lng": 0,
      "Time": "0001-01-01T00:00:00Z",
      "FixValidity": 0,
      "PressureAltitude": 0,
      "GNSSAltitude": 0,
      "IData": null,
      "NumSatellites": 0,
      "Description": ""
    },
    "Turnpoints": null,
    "Finish": {
      "Lat": 0,
      "Lng": 0,
      "Time": "0001-01-01T00:00:00Z",
      "FixValidity": 0,
      "PressureAltitude": 0,
      "GNSSAltitude": 0,
      "IData": null,
      "NumSatellites": 0,
      "Description": ""
    },
    "Landing": {
      "Lat": 0,
      "Lng": 0,
      "Time": "0001-01-01T00:00:00Z",
      "FixValidity": 0,
      "PressureAltitude": 0,
      "GNSSAltitude": 0,
      "IData": null,
      "NumSatellites": 0,
      "Description": ""
    },
    "Description": ""
  },
  "DGPSStationID": "",
  "Signature": ""
}